lib/lrama/states/item.rb

```diff
@@ -56,11 +56,11 @@ def new_by_next_position
         Item.new(rule: rule, position: position + 1)
       end
 
-      def symbols_before_dot
+      def symbols_before_dot # steep:ignore
```

Comment:
`Array#[]` is nullable but `symbols_before_dot` and `symbols_after_dot` are not.
But I couldn't make `Array#[]` temporarily not null, so I added `steep:ignore` .